From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Wolfgang Bumiller <w.bumiller@proxmox.com>
Date: Wed, 2 Jan 2019 14:37:58 +0100
Subject: [PATCH] PVE: [Config] Disable lxc.monitor cgroup

When not using relative cgroups this makes lxc unusable
within systemd service files as the service cgroup becomes
empty.

Signed-off-by: Wolfgang Bumiller <w.bumiller@proxmox.com>
---
 src/lxc/start.c | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/src/lxc/start.c b/src/lxc/start.c
index 1cf792aa..24f387de 100644
--- a/src/lxc/start.c
+++ b/src/lxc/start.c
@@ -1954,15 +1954,15 @@ int __lxc_start(const char *name, struct lxc_handler *handler,
 		goto out_fini_nonet;
 	}
 
-	if (!cgroup_ops->monitor_create(cgroup_ops, handler)) {
-		ERROR("Failed to create monitor cgroup");
-		goto out_fini_nonet;
-	}
-
-	if (!cgroup_ops->monitor_enter(cgroup_ops, handler->monitor_pid)) {
-		ERROR("Failed to enter monitor cgroup");
-		goto out_fini_nonet;
-	}
+	//if (!cgroup_ops->monitor_create(cgroup_ops, handler)) {
+	//	ERROR("Failed to create monitor cgroup");
+	//	goto out_fini_nonet;
+	//}
+
+	//if (!cgroup_ops->monitor_enter(cgroup_ops, handler->monitor_pid)) {
+	//	ERROR("Failed to enter monitor cgroup");
+	//	goto out_fini_nonet;
+	//}
 
 	if (geteuid() == 0 && !lxc_list_empty(&conf->id_map)) {
 		/* If the backing store is a device, mount it here and now. */
-- 
2.11.0

